n3fit/src/evolven3fit_new/eko_utils.py

```diff
@@ -1,13 +1,15 @@
 from ekobox import gen_theory, gen_op
 from eko import run_dglap
 
+from validphys.api import API 
 from validphys.loader import Loader
 from . import utils
 
 
 def construct_eko_cards(theoryID, op_card_dict, t_card_dict, q_fin, q_points, x_grid):
     """Return the theory and operator cards used to construct the eko"""
     # theory_card construction
+    _ = API.theoryid(theoryid = theoryID)
```

Comment:
If you are already using the API to get the theory I don't think you need the loader anymore.